updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+This is because `oc` command enhancement is not available in previous releases.
+====
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+[id="update-airgapped-env"]
+== Updating an air gapped {product-title} cluster
+
+An air gapped environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap. It may also be the case that there are multiple clusters within the air gapped network. In such a case it makes sense to configure a central registry in the air gapped environment from which every cluster can pull required images for upgrade.
+
+.Procedure
+
+. Findout if there is link:https://access.redhat.com/solutions/4583231[an update path] exists between your current version and the version you want the cluster to move.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/ocp-release:4.3.0-x86_64
+----
++
+
+** Take the media to the disconnected environment and upload the images to the local container registry
++
+----
+$ oc image mirror "--from-dir=${REMOVABLE_MEDIA_PATH}/mirror" file://openshift/release:4.3.0* REGISTRY/REPOSITORY
+----
++
+. Use `oc` to log in to a given cluster to be upgraded.
++
+[NOTE]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
+
+This is because `oc` command enhancement is not available in previous releases.
+====
++
+. Apply the the mirrored release image signature ConfigMap to the connected cluster
++
+----
+$ oc apply -f mirror/config/signature-sha256-81154f5c03294534.yaml
+----
+. Perform the upgrade
++
+----
+$ oc adm upgrade --allow-explicit-upgrade --to-image REGISTRY/REPOSITORY@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+----
+
+[id="update-disconnected-manual_method_configmap_creation"]
+== Manual method to create the image signature configmap
+
+The following commands will create and apply the image signature configmap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ export OCP_RELEASE_NUMBER=<4.y.z>
+----
++
+. Get the release image digest from link:https://quay.io/[Quay].
++
+----
+$ export DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-x86_64 | sed -n 's/Pull From: .*@//p')"
```

Comment:
should we paramaterize the arch too, instead of hard-coding `-x86_64`?  I really need to get back around to fixing [this][1], so we can stop talking about tags in these docs...

[1]: https://github.com/openshift/openshift-docs/pull/19266#discussion_r375878575